src/sources/docker_logs.rs

```diff
@@ -318,8 +318,6 @@ struct DockerLogsSource {
     main_recv: mpsc::UnboundedReceiver<Result<ContainerLogInfo, ContainerId>>,
     /// It may contain shortened container id.
     hostname: Option<String>,
-    /// True if self needs to be excluded
-    exclude_self: bool,
```

Comment:
Could we leave this, but deprecate it and output a warning if it is specified? That would allow us to more smoothly transition anyone using it.

Reply:
This was not a configurable option so I don't think this is an option.

Reply:
Previous behaviour is: if any of the `include_*` options is set, disable `exclude_self`.

Reply:
Ah! Yes, I see.